apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -98,23 +97,6 @@ const Courses = React.createClass({
           ref="flashes"
         />
 
-        {!isTeacher && (
-          <ProtectedStatefulDiv
-            style={styles.userHero}
-            ref="userHero"
```

Comment:
I think so, yes, because the same info will be handled with the StudentTopCourse on the student homepage.